### DISCUSSION TITLE
Audacious window is always centered on Desktop with Wayland

### DISCUSSION THREAD

**Discussion by warcayac** [audacious-media-player/audacious#126] [Q&A] [answered]
my info:
```
OS: EndeavourOS Linux x86_64
Kernel: 6.8.2-zen2-1-zen
Shell: zsh 5.9
Resolution: 3440x1440
DE: Plasma 6.0.2
WM: kwin
Memory: 15905MiB
```

In X11 Audacious remembers the last position of its window but with Wayland it always appears centered on the desktop. Any fix?

**Comment by jlindgren90** [comment, marked as answer]
Wayland does not allow applications to position their own windows. There is a new protocol being worked on to allow this (https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/264) but it will be a while before toolkits support it and applications can use it.

You can run Audacious under XWayland in the meantime. Set `QT_QPA_PLATFORM=xcb` in your environment.
